action.yaml

```diff
@@ -163,12 +165,12 @@ runs:
           ${{ env.GITHUB_EVENT_PULL_REQUEST_HEAD_SHA || github.event.pull_request.head.sha }}
         GITHUB_EVENT_PULL_REQUEST_NUMBER:
           ${{ env.GITHUB_EVENT_PULL_REQUEST_NUMBER || github.event.pull_request.number }}
-        GITHUB_REF_NAME: ${{ env.GITHUB_REF_NAME || github.ref_name }}
-        GITHUB_REPOSITORY: ${{ env.GITHUB_REPOSITORY || github.repository }}
-        GITHUB_TOKEN: ${{ github.token }}
         INPUT_ARGUMENTS: ${{ inputs.arguments }}
+        INPUT_CHECK_RUN_ID: ${{ inputs.check-run-id }}
+        INPUT_GITHUB_REF_NAME: ${{ env.INPUT_GITHUB_REF_NAME || github.ref_name }}
```

Comment:
should this be 
`INPUT_GITHUB_REF_NAME: ${{ env.GITHUB_REF_NAME || github.ref_name }}`

To my knowledge, we don't have a whole lot of coverage of specifying different input values, so just want to make sure we get this right...